react/features/participants-pane/components/styled.js

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Icon, IconHorizontalPoints } from '../../base/icons';
+import { ActionTrigger } from '../constants';
+
+export const ignoredChildClassName = 'ignore-child';
+
+export const AntiCollapse = styled.br`
+  font-size: 0;
+`;
+
+export const Button = styled.button`
+  align-items: center;
+  background-color: ${
+    // eslint-disable-next-line no-confusing-arrow
+    props => props.primary ? '#0376da' : '#2a3a4b'
+};
+  border: 0;
+  border-radius: 3px;
+  display: flex;
+  font-weight: unset;
+  justify-content: center;
+`;
+
+export const Container = styled.div`
+  box-sizing: border-box;
+  flex: 1;
+  overflow-y: auto;
+  position: relative;
+  padding: 0 ${props => props.theme.panePadding}px;
+  width: 315px;
+
+  & > * + *:not(.${ignoredChildClassName}) {
+    margin-top: 16px;
+  }
+
+  &::-webkit-scrollbar {
```

Comment:
I don't think this works in Firefox. Is it OK to be used?

Reply:
Firefox should be supported the same way we support Chrome.

Reply:
There are exiting customizations done in scss for webkit scrollbars that mess with layout, am just disabling for Chrome. Firefox scrollbars are OK.